css/index-flex1.css

```diff
@@ -270,14 +311,15 @@ margin-right: 10px;
 footer {
     width: 100%;
     background: black;
+    height: 100%;
```

Comment:
height is 100% of what container ? I'm not sure you need the height 100% here. Try it without and see what it does.